### PR TITLE
[airlock] Create containers

### PR DESCRIPTION
**Pull request by LizaShak**
# PR for issue

## What is being addressed

Create relevant blob containers on each airlock step.

## How is this addressed

- Create draft container 
- On each status update create target container before copy

### CODE REVIEW

File: airlock_processor/StatusChangedQueueTrigger/__init__.py [hunk 1]
```diff
@@ -47,9 +52,31 @@ def main(msg: func.ServiceBusMessage):
...
+    except KeyError as e:
+        logging.error(f'Missing environment variable: {e}')
+        raise
+
+    if new_status == "draft" and request_type == constants.IMPORT_TYPE:
+        account_name = constants.STORAGE_ACCOUNT_NAME_IMPORT_EXTERNAL.format(tre_id)
+        account_rg = constants.CORE_RESOURCE_GROUP_NAME.format(tre_id)
+        sa_connection = blob_operations.get_storage_connection_string(account_name, account_rg, storage_client)
+        blob_service_client = BlobServiceClient.from_connection_string(sa_connection.connection_string)
+        blob_operations.create_container(blob_service_client, req_id)
```
Comment (eladiw) [inline airlock_processor/StatusChangedQueueTrigger/__init__.py:66]: these 3 lines are identical to those on lines71-73
you can reuse it

**Review by eladiw — APPROVED** [review]
minor comment. approving but please fix

File: airlock_processor/shared_code/blob_operations.py [hunk 2]
```diff
@@ -1,11 +1,67 @@
...
+def get_storage_management_client():
+    try:
+        subscription_id = os.environ["AZURE_SUBSCRIPTION_ID"]
+    except KeyError as e:
+        logging.error(f'Missing environment variable: {e}')
+        raise
+
+    managed_identity = os.environ.get("MANAGED_IDENTITY_CLIENT_ID")
+    if managed_identity:
+        logging.info("using the Airlock processor's managed identity to get build storage management client")
```
Comment (eladiw) [inline airlock_processor/shared_code/blob_operations.py:59]:
```suggestion
        logging.info("using the Airlock processor's managed identity to get storage management client")
```
?

**Review by eladiw — APPROVED** [review]

File: airlock_processor/StatusChangedQueueTrigger/__init__.py (hunk 1 above)
Comment (LizaShak) [reply airlock_processor/StatusChangedQueueTrigger/__init__.py]: done

**Pull request merged**